src/gleam/iterator.gleam

```diff
@@ -421,7 +421,7 @@ fn do_flatten(flattened: fn() -> Action(Iterator(a))) -> Action(a) {
   }
 }
 
-/// Flattens an iterator of iterators, creating a new iterator.
+/// Joins an iterator of iterators into a single iterator.
```

Comment:
"join" is a different thing in Gleam, could you change it back to flattens please?

Reply:
Sure!